modular_zubbers/code/modules/clothing/mask/gasmask.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/clothing/mask/gas/modulator
+	name = "modified gas mask"
+	desc = "An older model of gas mask issued for use on station to help protect against airborne hazards. This one appears to be \
+		heavily modified, and the filter assembly has been replaced with a voice modulator to make the wearer sound more robotic."
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	max_filters = 0
+	///Our SPEAKING name while our voice is cloaked. Defaults to "Unknown", but can be set by var editing.
+	var/modulated_name = "Unknown"
+	///Used to store a special name if they wearer has one before equipping the mask.
+	var/previous_special_name = null
+	///If we are actually using the voice changer or not 
+	var/modulate_voice = TRUE 
+
+
+/obj/item/clothing/mask/gas/modulator/examine(mob/user)
+	. = ..()
+	. += span_notice("Its voice modulator is currently [modulate_voice ? "enabled" : "disabled"]. <b>Alt-click</b> to toggle it.")
```

Comment:
Would like this to be a context tooltip when you mouse over the item instead of in the examine text (the alt click to toggle part)